crates/shared/src/arguments.rs

```diff
@@ -295,6 +295,13 @@ pub struct Arguments {
     /// Override address of the balancer vault contract.
     #[clap(long, env)]
     pub balancer_v2_vault_address: Option<H160>,
+
+    /// Deprecate market orders (orders with positive signed fee) starting from
+    /// date
+    ///
+    /// TODO: remove this option after the deprecation date
```

Comment:
The `TODO` will show up in the `--help` text.

Reply:
Removed. Will add the task for removal.